### ISSUE TITLE
Multivalue attributes have single value

### ISSUE THREAD

**Issue by andriusdj** [kakwa/ldapcherry#12]
Hello! Almost all attributes in LDAP can have multiple values. E.g. Email attribute has many values: user@example.org, username@example.org, u@example.org, however only one (first) email is displayed in the form when editing the field. 

Is it possible to change the form to be able to edit multi value attributes? For example use <textarea> for mail and place each value into new line.

**Comment by kakwa** [comment]
Indeed, it's something I've kind of ignored because it was not completely trivial to handle.

Also, I'm not sure what would happen if modifying the only emailed displayed, maybe it will through an error, or it will modify one of the three email at random, or replace all the 3 emails by the one registered.

I'm not a big fan, "textarea, line by line" option, it's not really intuitive and explicit.

I see 2 options that could be valid:
* a '+' sign on the field that adds another input field of the same type (and a '-' to remove one of the fields
* being able to declare several time the same backend attribute, you then just add an email1, email2 and email3 field with the same attribute, making the email2 and email3 optional.

The second option is probably the simplier to implement, but it only support a max set of attributes. The first one is more tricky to implement, maybe a bit less intuitive on the UI, but a bit more flexible (you could have users with 42, 69, or any number of emails... not sure if it's a good idea though ^^).

Both might get tricky to implement because of the multi-backend, specially if one backend authorize a multi valued field, and the other don't.

**Comment by vishwakumba** [comment]
This feature could be useful to us. We have a multi-value field that supports up to 5 values. We are unable to modify this field due to this missing enhancement.

**Comment by kagareu** [comment]
I have the same issue with ProxyAddress to be used on O365 Sync.
